.pre-commit-config.yaml

```diff
@@ -38,7 +38,8 @@ repos:
         language: python
         additional_dependencies: [pygments, restructuredtext_lint]
     -   id: changelogs-rst
-        name: changelog files must end in .rst
-        entry: ./scripts/fail
-        language: script
-        files: 'changelog/.*(?<!\.rst)$'
+        name: changelog filenames
+        language: fail
```

Comment:
note here that `language: fail` is a new [pre-commit "language"](https://pre-commit.com/#fail)